coderadar-rest/src/test/java/io/reflectoring/coderadar/rest/unit/useradministration/RegisterUserControllerTest.java

```diff
@@ -24,9 +25,11 @@ void registerUserWithIdOne() {
 
     Mockito.when(registerUserUseCase.register(command)).thenReturn(1L);
 
-    ResponseEntity<IdResponse> responseEntity = testSubject.register(command);
+    ResponseEntity<Object> responseEntity = testSubject.register(command);
 
+    Assertions.assertTrue(responseEntity.getBody() instanceof IdResponse);
+    IdResponse responseBody = (IdResponse) responseEntity.getBody();
     Assertions.assertEquals(HttpStatus.CREATED, responseEntity.getStatusCode());
-    Assertions.assertEquals(1L, responseEntity.getBody().getId());
```

Comment:
`responseBody` wird nur ein einziges mal verwendet, daher sehe ich hier keinen Mehrwert darin, das in einer eigenen Variablen zu speichern.